tests/integration/hooks.bats

```diff
@@ -14,6 +14,7 @@ function setup() {
 
 	teardown_debian
 	setup_debian
+	update_config '.root.readonly |= false'
```

Comment:
as I noted earlier, it should be done
 - in a very test case that requires it
 - as a separate patch describing why it is done (since the linker in hooks test likes to create files)

Reply:
If this PR don't have this patch, CI verification failed. Can  I add some notes ?

Reply:
I skiped the hook test case, but add some TODO notes for an new PR to fix this issue.